.jscsrc

```diff
@@ -1,3 +1,4 @@
 {
-  "preset": "google"
+  "preset": "google",
+  "requireCamelCaseOrUpperCaseIdentifiers": "ignoreProperties"
```

Comment:
because the object coming from the REST API is not camel case and I unless I change it, I cannot fix this